src/index.html

```diff
@@ -9,10 +9,56 @@
     <title>Stars</title>
     <link
```

Comment:
The attributes of the <link> tag should start each one on a new line with 2-space indentation relative to the tag. The tag’s closing bracket should be on the same level as the opening one.